python/mxnet/symbol.py

```diff
@@ -1111,12 +1111,227 @@ def _get_ndarray_inputs(arg_key, args, arg_names, allow_missing):
             raise TypeError('Only accept list of NDArrays or dict of str to NDArray')
         return c_array(NDArrayHandle, arg_handles), arg_arrays
 
-    def simple_bind(self, ctx,
-                    grad_req='write',
-                    type_dict=None,
-                    group2ctx=None,
-                    **kwargs):
-        """Binds current symbol to get an executor, allocate all the arguments needed.
+    def simple_bind(self, ctx, grad_req='write', type_dict=None, group2ctx=None,
+                    param_names=None, shared_exec=None, shared_data_arrays=None, **kwargs):
+        """This function is DEPRECATED.
```

Comment:
oops, maybe I accidentally copied the description of simple_bind_v1 and forgot to delete deprecation statement.